Source/Examples/README.md

```diff
@@ -1,49 +1,73 @@
-   
-### RMX410x Configuration:
-   - A new Instrument Addon is created in the target System Definition File, with an RS-xxx protocol type.
-   - Configuration, Control and Feedback groups are created.
-   - Messages configured with MetaData and Data channels are created.
-   - The Messages for controlling and getting feedback from the RMX410x device, and Channels that control the output state and voltage levels are configured.
-   - [RMX410x User manual](https://www.ni.com/pdf/manuals/377396b.pdf)
-   
-## **Instructions on how to run the examples:**
-
-1. Open the Instrument Addon Scripting API Examples.lvproj
-2. Open the desired VI and modify the message type, settings, config file paths as desired.
-3. Run the VI
-
-**Note:** It is strongly advised to operate the modifications mentioned in step number 2, as the logic in the VIs is for demonstration purposes only. You may need another wait type for your Response, other Channel types, padding may not make sense in your use case, etc.
\ No newline at end of file
+### Basic Example
+
+`Basic Example.vi` configures an Instrument Addon with a single message, both Command and Response, and exports the configuration to a file on disk.
+
+The VI executes the following steps:
+   1. Creates a new TCP Protocol object and uses the protocol as an input for a new Instrument Addon Custom Device.
+   1. Updates the name of the add-on.
+   1. Creates a message section and a new message.
+   1. Adds and configures the Command and Response sections.
+
+
+   **Note:** The **settings** input for the **Configure Response** uses a known value, 42, as the channel refnum for the linked channel. This channel is composed of a variable name and a channel refnum.
+
+### Alter the IP Address of a Protocol
+
+`Alter the IP Address of a protocol.vi` alters the IP Address of a protocol while leaving other properties unchanged.
+
+The VI executes the following steps:
+   1. Uses the input refnum to fetch the protocol of the Instrument Addon Custom Device.
+   1. Determines if the protocol is **TCP** or **AK TCP**.
+   1. Alters the IP Address and inputs the new value to the appropriate protocol factory.
+
+The factory produces a protocol object that is used as an input for the **Set Protocol** node.
+
+### Setting a new Protocol, Regardless of Previous Configuration
+
+### RMX410x Configuration
+
+`RMX410x.vi` executes the following steps:
+   1. Creates a new Instrument Addon in the target system definition file with an RS-xxx protocol type.
+   1. Creates **Configuration**, **Control**, and **Feedback** groups.
+   1. Configures messages with created MetaData and Data channels.
+   1. Configures the messages for controlling and getting feedback from the RMX410x device
+   1. Configures the channels that control the output state and voltage levels.
+
+For more information on the RMX410x, refer to the [RMX410x User manual](https://www.ni.com/pdf/manuals/377396b.pdf).
+
```

Comment:
There appears to be another VI called 'Setting a New Protocol' that we do not provide information on. Should we?

Reply:
This example is a very simple one that sets a TCP protocol to an existing Instrument Addon. So the information we can provide is a bit redundant, as we can just say:

Setting a TCP protocol to an existing Instrument Addon custom device.

Reply:
I'm fine leaving it out then. Just wanted to make sure we were not missing anything.